### ISSUE TITLE
Pool + `conn.create_scalar_function`

### ISSUE THREAD

**Issue by jessekrubin** [ryanfowler/async-sqlite#12]
Creating/registering sqlite functions/extensions with `Pool` only registers for one connection.

**Comment by jessekrubin** [comment]
@ryanfowler this problemo extends to `attach` as well so attaching from a pool only attaches on one client.

**Comment by ryanfowler** [comment]
Yeah, one of the problems with `Pool` right now is that you can't run a change on _all_ connections in the pool. I'll need to add a method to `Pool` that does that. Something like an `all` method that takes an [`Fn`](https://doc.rust-lang.org/nightly/core/ops/trait.Fn.html) that it applies to every connection in the pool.

I'll hopefully have some time soon to work on that

**Comment by jessekrubin** [comment]
@ryanfowler I would be happy to give it a go (if I have the time soon). 

BTW If you wanna see your lib in the wild, you can checkout this: https://github.com/jessekrubin/utiles/blob/main/crates/utiles/src/sqlite/async_sqlite3.rs and https://github.com/jessekrubin/utiles/blob/main/crates/utiles/src/utilesqlite/mbtiles_async_sqlite.rs  -- Any thoughts/ideas appreciated!

**Comment by ryanfowler** [comment]
That would be great!

**Comment by ryanfowler** [comment]
Closed by https://github.com/ryanfowler/async-sqlite/pull/21
